<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_PointInfo</class>
 <widget class="QDialog" name="Dialog_PointInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>177</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>映射点信息</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_xi">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_yi">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_zi">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Z：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>当前映射点信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_x">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_y">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>像点坐标：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_xyzi">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>映射点坐标：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>放弃</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
